bundles/org.eclipse.cdt.lsp.clangd/src/org/eclipse/cdt/lsp/internal/clangd/ClangdPreferredOptions.java

```diff
@@ -110,7 +113,9 @@ public List<String> toList() {
 		}
 		list.add(NLS.bind("--query-driver={0}", queryDriver())); //$NON-NLS-1$
 
-		list.addAll(additionalOptions());
+		var additionalOptions = additionalOptions();
+		if (!additionalOptions.isEmpty())
```

Comment:
is this necessary? `addAll` will do nothing for empty list, isn't it?

Reply:
Thank you. It's not necessary.